### PR TITLE
Update components with UIB feedback

### LINKED ISSUE #259: Investigate & fix font weight
### Description
For some reason the impact of changing the font weight in our app is very miniscule.

Should be timeboxed.

In App
image

Expected (in Figma)
image

### Acceptance Criteria

  - [ ] Font weight should have a noticeable effect, as shown in Figma

### LINKED ISSUE #284: Update the favicon
### Description

Update the favicon to be the EDD image a la https://edd.ca.gov/ 

### Acceptance Criteria

- [ ] Correct favicon shows in staging

### LINKED ISSUE #287: Update FE Components with UIB Feedback
### Description

 - Drop Alert banner (Close #258) 
 - Remove box surrounding Next Steps
   - Tweak spacing accordingly
 - Set claim status info line to third level header
 - Unbold the headers

### Acceptance Criteria

  - [ ] components created according to the design specifications
  - [ ] Work with Design for last 10%

### PR DESCRIPTION
**Pull request by lomky**
This is ready for review, but a caveat! We don't have the section under claim status with more details, and we do not have the split next steps. I did a rough pass with placeholders but will plan on doing a final one after the logic is properly tied in from the back. For now don't mind spacing around that.

Compare to the UIB Content Updates under the SP9 6/23 of Figma.

===

Resolves #287 
Resolves #284 
Resolves #259 

- [x] Final pass done pairing with @nguyen-tina 
- [x] Components created according to the design specifications
- [x] Favicon matches EDD
- [x] Bold is boldy
- [ ] Changes are confirmed on Staging post-merge into `main`

### CODE REVIEW

File: components/ClaimDetails.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const ClaimDetails: React.FC<ClaimDetailsProps> = ({
   }
 
   return (
-    <div className="claim-details container">
+    <div className="claim-details claim-section container">
```
Comment (lomky) [inline components/ClaimDetails.tsx:31]: Part of a spacing refactor!

File: components/ClaimStatus.tsx [hunk 2]
```diff
@@ -12,19 +12,12 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({ loading = false, statu
...
       <h2>{t('claim-status.title')}</h2>
-      <div className="pending-status">
-        <TextLine loading={loading} text={t(statusDescription)} />
-      </div>
-      <div className="status-box">
-        <div className="topbar">
-          <h3 className="next-steps">{t('claim-status.next-steps')}</h3>
-        </div>
-        <div className="explanation">
-          <NextSteps loading={loading} nextSteps={nextSteps} />
```
Comment (lomky) [inline components/ClaimStatus.tsx:25]: Refactored into a new component, in prep for there being two of them!

File: components/NextSteps.tsx [hunk 3]
```diff
@@ -2,23 +2,41 @@ import { Shimmer } from './Shimmer'
...
+    'a very very very very very very very very very very very very very very very very very very very very very very very very very very very long bullet',
+    'another step with longer text',
+    'a third bullet',
+    'another very very very very very very very very very very very very very very very very very very very very very very very very very very very long other bullet',
+  ],
+}) => {
+  let loadableContent: JSX.Element
   if (loading) {
-    return <Shimmer width="100%" height={50} baseColor="#B6B2B2" shimColor="#656565" borderRadius={3} />
+    loadableContent = <Shimmer width="100%" height={50} baseColor="#B6B2B2" shimColor="#656565" borderRadius={3} />
```
Comment (lomky) [inline components/NextSteps.tsx:21]: Only hide the original scope of the component, tho

File: pages/index.tsx [hunk 4]
```diff
@@ -74,8 +73,8 @@ export default function Home({
         <title>{t('title')}</title>
         <link rel="icon" href="/claimstatus/favicon.ico" />
         <link href="https://fonts.googleapis.com/css?family=Source Sans Pro" rel="stylesheet" />
+        <link href="https://allfont.net/allfont.css?fonts=source-sans-pro-bold" rel="stylesheet" type="text/css" />
```
Comment (lomky) [inline pages/index.tsx:76]: For _real_ bold

File: styles/globals.scss [hunk 5]
```diff
@@ -87,62 +84,44 @@ body {
...
-    margin-bottom: 3rem;
+  .claim-section {
+    margin-top: 3rem;
   }
 
-  .title {
-    margin-bottom: 0.5rem;
+  .claim-subsection {
+    margin-top: 1.5rem;
   }
```
Comment (lomky) [inline styles/globals.scss:95]: Moving our higher order spacing out to shared class names

File: tests/pages/index.test.tsx [hunk 6]
```diff
@@ -40,6 +40,7 @@ describe('Main component shows loading', () => {
     render(<Index loading scenarioContent={scenarioContent} />)
     expect(screen.queryByText('Claim Tracker')).toBeInTheDocument()
     expect(screen.queryByText('Claim Status')).toBeInTheDocument()
+    expect(screen.queryByText('There are no pending issues at this time.')).not.toBeInTheDocument()
```
Comment (lomky) [inline tests/pages/index.test.tsx:43]: Check the header-that-is-content hides when loading

File: stories/ClaimStatus.stories.tsx [hunk 7]
```diff
@@ -11,6 +11,6 @@ const Template: Story<ClaimStatusProps> = (args) => <ClaimStatusComponent {...ar
 
 export const ClaimStatus = Template.bind({})
 ClaimStatus.args = {
-  statusDescription: 'claim-status:scenarios:scenario1.description',
+  statusDescription: 'claim-status:scenarios.scenario1.description',
```
Comment (rocketnova) [inline stories/ClaimStatus.stories.tsx:14]: Thank you!

**Review by rocketnova — APPROVED** [review]
LGTM!

Note: I did not review any of the design styling and am assuming that you have/will go over that with @nguyen-tina.

**Comment by nguyen-tina** [comment]
Fine to keep this styling rough, since you'll be doing another pass later!

**Pull request merged**